solution1/crd.yaml

```diff
@@ -0,0 +1,26 @@
+# 
+# NOTE: This file is not used in the solution this serves an example how the yaml
+# file looks like. Content of this file is copied crdJson variable in JSON
+# format.
```

Comment:
Having the yaml file is nice <3 Nitpick but I'd rewrite the comment like this.

This file is not used in the solution. It is an example of what the YAML file looks like. Content in this file is copied from the crdJson JSON variable.

Reply:
Done.